pageserver/src/layered_repository/blob.rs

```diff
@@ -1,46 +1,17 @@
-use std::io::{Read, Write};
 use std::os::unix::prelude::FileExt;
```

Comment:
This file has become trivial, maybe we should get rid of it and move the remaining code into `delta_layer.rs`